random.js

```diff
@@ -0,0 +1,13 @@
+const randomizeButoon = document.querySelector('#randomize');
+// create callback function to 'button click'
+const RandomImg = function()
+ {
+    // Create new img element and give it a src property
+    const img = document.createElement('img');
+    img.setAttribute('src', 'https://source.unsplash.com/random/800X600')
```

Comment:
one way to do the bonus is to get the height and whidth value from the DOM and assign it to a variable. 

then use those variables in the URL, similar to this
```
let width = document.querySelector("input.width");
let height = document.querySelector("input.height");
let url = ("https://source.unsplash.com/" + width + "x" + height);
let image = document.querySelector(".img");
image.setAttribute("src", url);
```